front/src/utils/date.ts

```diff
@@ -28,7 +32,14 @@ export function dateTimeFormatting(date: Date, withoutTime: boolean = false) {
   }
   // Force interpreting the date in UTC
   const dateToUTC = dayjs(date).utc(true);
-  const dateFormat = withoutTime ? 'D MMM YYYY' : 'D MMM YYYY HH:mm';
+  let dateFormat;
+  if (formatType === 'default') {
+    dateFormat = withoutTime ? 'D MMM YYYY' : 'D MMM YYYY HH:mm';
+  } else if (formatType === 'alternate') {
+    dateFormat = withoutTime ? 'DD/MM/YY' : 'DD/MM/YY HH:mm';
+  } else {
+    throw new Error('Invalid format type');
+  }
```

Comment:
I think that for this case, it would be better to use a switch:
```suggestion
  switch (formatType) {
    case 'default':
      dateFormat = withoutTime ? 'D MMM YYYY' : 'D MMM YYYY HH:mm';
      break;
    case 'alternate':
      dateFormat = withoutTime ? 'DD/MM/YY' : 'DD/MM/YY HH:mm';
      break;
    default:
      throw new Error('Invalid format type');
  }
```

Reply:
fix